libraries/protocol/include/scorum/protocol/operations.hpp

```diff
@@ -60,14 +60,20 @@ using operation = fc::static_variant<vote_operation,
                                      comment_payout_update_operation,
                                      comment_reward_operation,
                                      curation_reward_operation,
-                                     fill_vesting_withdraw_operation,
                                      hardfork_operation,
                                      producer_reward_operation,
                                      active_sp_holders_reward_operation,
                                      return_scorumpower_delegation_operation,
                                      shutdown_witness_operation,
                                      witness_miss_block_operation,
-                                     expired_contract_refund_operation>;
+                                     expired_contract_refund_operation,
+                                     acc_total_vesting_withdraw_operation,
```

Comment:
acc_total_vesting_withdraw_operation and devpool_total_vesting_withdraw_operation actually means acc_vesting_withdraw_finish_operation and devpool_vesting_withdraw_finish_operation. Could you rename it?